components/CommentForm/index.js

```diff
@@ -1,23 +1,25 @@
 import styled from "styled-components";
 import { useRouter } from "next/router";
 
-export default function CommentForm({handleSave}) {
+export default function CommentForm({ handleSave }) {
   const router = useRouter();
 
   return (
     <StyledForm onSubmit={handleSave}>
       <label htmlFor="comment">Comment</label>
-      <textarea type="text" name="comment" id="comment" rows="5"></textarea>
+      <textarea name="comment" id="comment" rows="5"></textarea>
       <button type="submit">Save Player</button>
     </StyledForm>
   );
 }
 
 const StyledForm = styled.form`
-  margin: 10% 0% 10% 20%;
   display: flex;
   flex-direction: column;
   justify-content: space-evenly;
   width: 200px;
   height: 150px;
+  position: fixed;
+  top: 25%;
+  left: 20%;
```

Comment:
This will lead to unexpected behaviour. Try it out in your browser: If the viewport height gets small it will appear above the other stuff. I cannot see any advantage of position: fixed.